lib/matplotlib/backends/backend_pdf.py

```diff
@@ -550,7 +550,7 @@ def close(self):
                                for val in six.itervalues(self.alphaStates)]))
         self.writeHatches()
         self.writeGouraudTriangles()
-        xobjects = dict(six.itervalues(self.images))
+        xobjects = dict(x[1:] for x in six.itervalues(self.images))
```

Comment:
All usages have been updated (there's only 3).  I really don't think we should consider the PDF backend part of public API, and particularly not the `images` member.  This is definitely one of those cases that predates the private member convention in Python that just should have been private all along.